apollo-router/tests/snapshots/apollo_reports__batch_send_header.snap

```diff
@@ -1088,6 +1088,7 @@ traces_per_query:
     stats_with_context: []
     referenced_fields_by_type: {}
     internal_traces_contributing_to_stats: []
+    query_metadata: ~
```

Comment:
This maps expectedly to the updates brought in by the `QueryMetadata` type in the `reports.proto` update itself. ✔️